packages/ember-intl/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {
     "@formatjs/icu-messageformat-parser": "^2.7.6",
-    "@formatjs/intl": "^2.10.0",
+    "@formatjs/intl": "^2.10.1",
```

Comment:
Only a version bump, safe to update.

https://github.com/formatjs/formatjs/blob/%40formatjs/intl%402.10.1/packages/intl/CHANGELOG.md#2101-2024-03-24